lib/tasks/migrations.rake

```diff
@@ -63,11 +63,22 @@ namespace :migrations do
   desc "Run uncompleted account deletions"
   task run_account_deletions: :environment do
     if AccountDeletion.uncompleted.count > 0
-      puts "Running account deletions.."
-      AccountDeletion.uncompleted.find_each(&:perform!)
+      puts "Running account deletions..."
+      AccountDeletion.uncompleted.find_each do |account_deletion|
+        print "Deleting #{account_deletion.person.diaspora_handle} ..."
+        progress = Thread.new {
+          loop {
+            sleep 10
+            print "."
+          }
+        }
+        account_deletion.perform!
+        progress.kill
+        puts " Done"
+      end
       puts "OK."
     else
-      puts "No acccount deletions to run."
+      puts "No account deletions to run."
```

Comment:
But... the hidden phun...!

Reply:
I copy pasted that from an older rake task ... and while adding the progress indicators the spell-checker of my IDE told me that there is something wrong here ;)